plugins/hydra_submitit_launcher/NEWS.md

```diff
@@ -0,0 +1,8 @@
+1.1 (2021-01-30)
```

Comment:
Could we add documentation on how to generate `NEWS.md` [here](https://hydra.cc/docs/next/development/documentation#news-entries) for both `core` and the plugins?

Reply:
Probably this is a better [place](https://hydra.cc/docs/next/development/release) ?

Reply:
yes, this is better :)